main.nf

```diff
@@ -1984,25 +1979,30 @@ process get_software_versions {
     echo $workflow.manifest.version &> v_pipeline.txt
     echo $workflow.nextflow.version &> v_nextflow.txt
     fastqc --version &> v_fastqc.txt 2>&1 || true
-    multiqc --version &> v_multiqc.txt 2>&1 || true
+    AdapterRemoval --version  &> v_adapterremoval.txt 2>&1 || true
+    fastp --version &> v_fastp.txt 2>&1 || true
     bwa &> v_bwa.txt 2>&1 || true
+    circulargenerator --help | head -n 1 &> v_circulargenerator.txt 2>&1 || true
     samtools --version &> v_samtools.txt 2>&1 || true
-    AdapterRemoval -version  &> v_adapterremoval.txt 2>&1 || true
-    picard MarkDuplicates --version &> v_markduplicates.txt  2>&1 || true
     dedup -v &> v_dedup.txt 2>&1 || true
-    preseq &> v_preseq.txt 2>&1 || true
-    gatk BaseRecalibrator --version 2>&1 | grep Version: > v_gatk.txt 2>&1 || true
-    vcf2genome &> v_vcf2genome.txt 2>&1 || true
-    fastp --version &> v_fastp.txt 2>&1 || true
-    bam --version &> v_bamutil.txt 2>&1 || true
+    picard MarkDuplicates --version &> v_markduplicates.txt  2>&1 || true
     qualimap --version &> v_qualimap.txt 2>&1 || true
-    cat $json &> v_damageprofiler.txt 2>&1 || true 
-    multivcfanalyzer --help | head -n 1 &> v_multivcfanalyzer.txt 2>&1 || true
+    preseq &> v_preseq.txt 2>&1 || true
+    gatk --version 2>&1 | head -n 1 > v_gatk.txt 2>&1 || true
+    freebayes --version &> v_freebayes.txt 2>&1 || true
     bedtools --version &> v_bedtools.txt 2>&1 || true
-    malt-run --help |& tail -n 3 | head -n 1 | cut -f 2 -d'(' | cut -f 1 -d, &> v_malt.txt 2>&1 || true
+    damageprofiler --version &> v_damageprofiler.txt 2>&1 || true
+    bam --version &> v_bamutil.txt 2>&1 || true
+    pmdtools --version &> v_pmdtools.txt 2>&1 || true
     angsd -h |& head -n 1 | cut -d ' ' -f3-4 &> v_angsd.txt 2>&1 || true 
+    multivcfanalyzer --help | head -n 1 &> v_multivcfanalyzer.txt 2>&1 || true
+    malt-run --help |& tail -n 3 | head -n 1 | cut -f 2 -d'(' | cut -f 1 -d ',' &> v_malt.txt 2>&1 || true
+    MaltExtract --help | head -n 2 | tail -n 1 &> v_maltextract.txt 2>&1 || true
+    multiqc --version &> v_multiqc.txt 2>&1 || true
 
-    java -jar ${jar} --version &> v_gatk3_5.txt 2>&1 || true 
+    ## Hardcoded as no --version flag or equivalent
+    echo "v1.1" > v_sexdeterrmine.txt
```

Comment:
We should make @TCLamnidis add such a flag!

Reply:
https://github.com/TCLamnidis/Sex.DetERRmine/issues/7 ;)